docs/index.adoc

```diff
@@ -12,33 +11,61 @@ This documentation is available in {pages}[HTML] and {pages}/ebook.pdf[PDF] form
 Software is available from the {project}[Korrel8r project].
 ====
 
-include::description.adoc[]
+[id="index-overview"]
+== Korrel8r overview
+
+Many observability tools and observable systems use their own unique data models, storage technologies, query languages, and related nomenclature.
+
+The differences between these nomenclatures can make it difficult for users to recognize relationships between different Kubernetes cluster resources. As a result, troubleshooting and identifying resources affected by issues in your cluster can be time consuming and difficult.
+
+Korrel8r is a correlation engine for observable signals that relates Kubernetes objects to different types of signal data. Given a _start_ object, Korrel8r searches for a chain of rules to related _goal_ objects. Korrel8r can also show the _neighbourhood_ of all data related to an object of interest.
+
+A REST API can be used by clients to make correlation queries. The REST API can be deployed as a service in a cluster, or run outside the cluster and connect to signal stores inside the cluster.
+
+Signal data can be stored in separate signal stores, and encoded using different schema.
+
+[IMPORTANT]
+====
+To navigate between correlated data, Korrel8r needs to access multiple data stores in the cluster.
+These include the Kubernetes API server (resouces), LokiStack (logs), Prometheus (metrics), and AlertManager (alerts).
+The initial release can only be deployed and used by an admin user that has full access to all these stores.
+
+See also link:{project}/issues/73[]
+====
+
+[id="index-kube-lineage"]
+=== Is kube-lineage the same as Korrel8r?
+
+link:https://github.com/tohjustin/kube-lineage[`kube-lineage`] is a CLI tool to display all the dependencies and dependents of a resource in a Kubernetes cluster.
+
+`kube-lineage` is similar to Korrel8r as both can follow chains of relationships between cluster resources.
+However, Korrel8r can also follow relationships with other types of observability data, such as logs, alerts or metrics. Korrel8r is designed to be extendable to new types of data and data stores in future, for example, network observability events.
```

Comment:
This not central to korrel8r. Somebody asked me about this tool and how it relates, so I stuck it in the FAQ, although in fairness it may just be an AQ.
```suggestion
```